internal/config/main.go

```diff
@@ -74,6 +75,7 @@ func New(opts ...Option) Config {
 		operatorOpAMPBridgeConfigMapEntry: defaultOperatorOpAMPBridgeConfigMapEntry,
 		logger:                            logf.Log.WithName("config"),
 		version:                           version.Get(),
+		enableJavaInstrumentation:         true,
```

Comment:
is there a reason we need to set this here if our default for the option is true?